unit_conversions.md

```diff
@@ -22,7 +22,7 @@ Table: Useful conversions for entering site, management, yield, and trait data [
 | DD\(^{\circ}\) MM' SS'' | XX.ZZZZ | \(\textrm{XX.ZZZZ} = \textrm{DD} + \textrm{MM}/60+\textrm{SS}/3600\) | to convert latitude or longitude from degrees, minutes, seconds to  decimal degrees |
 | lb | kg | \(Y=X\times 2.2\) | |
 | mm/s | \(\mu\) mol CO\(_2\) m\(^{2}\) s\(^{-1}\) | \(Y=X\times 0.04\) | |
-| m\(^2\) | ha | \(Y = X/10^6\) | |
+| m\(^2\) | ha | \(Y = X/10^5\) | |
```

Comment:
I don't usually look at BetyDB stuff anymore, but couldn't help noticing this: One hectare is 10^4 square meters, not 10^5.

Reply:
@gsrohde thanks for catching that ... fixed in e033609